cmd/loadtest/app.go

```diff
@@ -237,14 +241,18 @@ r - random modes
 7 - ERC721 mints
 v3 - UniswapV3 swaps
 R - total recall
-rpc - call random rpc methods`)
+rpc - call random rpc methods
+cc, contract-call - call a contract method`)
 	ltp.Function = LoadtestCmd.Flags().Uint64P("function", "f", 1, "A specific function to be called if running with `--mode f` or a specific precompiled contract when running with `--mode a`")
 	ltp.ByteCount = LoadtestCmd.Flags().Uint64P("byte-count", "b", 1024, "If we're in store mode, this controls how many bytes we'll try to store in our contract")
 	ltp.LtAddress = LoadtestCmd.Flags().String("lt-address", "", "The address of a pre-deployed load test contract")
 	ltp.ERC20Address = LoadtestCmd.Flags().String("erc20-address", "", "The address of a pre-deployed ERC20 contract")
 	ltp.ERC721Address = LoadtestCmd.Flags().String("erc721-address", "", "The address of a pre-deployed ERC721 contract")
 	ltp.ForceContractDeploy = LoadtestCmd.Flags().Bool("force-contract-deploy", false, "Some load test modes don't require a contract deployment. Set this flag to true to force contract deployments. This will still respect the --lt-address flags.")
 	ltp.RecallLength = LoadtestCmd.Flags().Uint64("recall-blocks", 50, "The number of blocks that we'll attempt to fetch for recall")
+	ltp.ContractAddress = LoadtestCmd.Flags().String("contract-address", "", "The address of the contract that will be used in `--mode contract-call`. This must be paired up with `--mode contract-call` and `--calldata`")
```

Comment:
I just discovered cobra's `MarkFlag` and `MarkFlags` methods and I think it's very handy, especially in such cases. 

<img width="584" alt="Screenshot 2023-12-05 at 09 23 47" src="https://github.com/maticnetwork/polygon-cli/assets/28714795/c1853841-2039-4dd2-a597-afb8546c5cd8">

You can mark those 3 flags required together using [`MarkFlagsRequiredTogether`](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsRequiredTogether). If the user specifies one or two of the three flag but not all three, it would throw an error.

Reply:
neat, good find! I'll try to utilize this soon :)